src/sound/GeneratedSoundBanks/Windows/SoundbanksInfo.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <SoundBanksInfo Platform="Windows" BasePlatform="Windows" SchemaVersion="12" SoundbankVersion="135">
 	<RootPaths>
-		<ProjectRoot>C:\MSFS SDK\Samples\WwiseSampleProject\</ProjectRoot>
-		<SourceFilesRoot>C:\MSFS SDK\Samples\WwiseSampleProject\.cache\Windows\</SourceFilesRoot>
-		<SoundBanksRoot>C:\MSFS SDK\Samples\WwiseSampleProject\GeneratedSoundBanks\Windows\</SoundBanksRoot>
+		<ProjectRoot>C:\Users\olive\Desktop\MSFSMODS\msfs-a320neo\src\sound\</ProjectRoot>
+		<SourceFilesRoot>C:\Users\olive\Desktop\MSFSMODS\msfs-a320neo\src\sound\.cache\Windows\</SourceFilesRoot>
+		<SoundBanksRoot>C:\Users\olive\Desktop\MSFSMODS\msfs-a320neo\src\sound\GeneratedSoundBanks\Windows\</SoundBanksRoot>
 		<ExternalSourcesInputFile></ExternalSourcesInputFile>
-		<ExternalSourcesOutputRoot>C:\MSFS SDK\Samples\WwiseSampleProject\GeneratedSoundBanks\Windows</ExternalSourcesOutputRoot>
+		<ExternalSourcesOutputRoot>C:\Users\olive\Desktop\MSFSMODS\msfs-a320neo\src\sound\GeneratedSoundBanks\Windows</ExternalSourcesOutputRoot>
```

Comment:
Same here

Reply:
it will just update when you press generate to your location of the project :)